### PR TITLE
refactor: importable module and event struct

### PR DESCRIPTION
**Pull request by mg98**
- Renamed module so that external projects can `go get github.com/trudi-group/ipfs-metric-exporter`
- Made `type event struct` public

### CODE REVIEW

File: metricplugin/rabbitmq.go [hunk 1]
```diff
@@ -207,15 +207,15 @@ func packagerLoop(logger *zap.SugaredLogger, msgIn <-chan event, packetsOut chan
 		}
 
 		(**currentBuffer).events = append((**currentBuffer).events, msg)
-		logger.Debugf("packager: got event %+v, now have %d events in buffer", msg, len((**currentBuffer).events))
+		logger.Debugf("packager: got Event %+v, now have %d events in buffer", msg, len((**currentBuffer).events))
```
Comment (mrd0ll4r) [inline metricplugin/rabbitmq.go:210]: leave this as it was, because it's an English sentence

File: metricplugin/rabbitmq.go [hunk 2]
```diff
@@ -128,14 +128,14 @@ func (p *RabbitMQPublisher) PublishConnectionEvent(ts time.Time, connected bool,
...
-	p.msgOut <- event{
+	p.msgOut <- Event{
 		Timestamp:      ts,
 		Peer:           peer.String(),
 		BitswapMessage: msg,
 	}
 }
 
-type event struct {
+type Event struct {
```
Comment (mrd0ll4r) [inline metricplugin/rabbitmq.go:138]: This will probably need a godoc comment :)

**Review by mrd0ll4r — CHANGES_REQUESTED** [review]
A few nitpicks

**Comment by mrd0ll4r** [comment]
Ah, and the checks have something to say

File: metricplugin/api.go [hunk 3]
```diff
@@ -67,7 +66,7 @@ type ConnectionEvent struct {
 	// The multiaddress of the remote peer.
 	Remote ma.Multiaddr `json:"remote"`
 
-	// The type of this event.
+	// The type of this Event.
```
Comment (mrd0ll4r) [inline metricplugin/api.go:69]: I wouldn't capitalize here, either. Sorry for missing this last time...

**Review by mrd0ll4r — CHANGES_REQUESTED** [review]

**Comment by mrd0ll4r** [comment]
`gofumpt` complains because imports should be grouped into stdlib and others, see [https://github.com/mvdan/gofumpt#added-rules](their added rules), especially the one about `std imports must be in a separate group at the top`.

**Comment by mrd0ll4r** [comment]
Fantastic, I think this should pass CI. Can you squash the commits into one, so I can merge?

**Comment by mg98** [comment]
Doesn't Github do it for you?

**Comment by mrd0ll4r** [comment]
It does, but I remember from somewhere that something about it is broken and I since stopped doing it... It will definitely break signing, which we don't use here, but it feels weird in general.

**Review by mrd0ll4r — APPROVED** [review]

**Pull request merged**